apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -284,7 +289,7 @@ export default class FormController extends React.Component {
         style={styles.pageButtons}
         items={this.getPageComponents().length}
         activePage={this.state.currentPage + 1}
-        onSelect={i => this.setPage(i - 1)}
+        onSelect={i => this.setPage(i - 1)} // eslint-disable-line react/jsx-no-bind
```

Comment:
unfortunately, it's not gonna work here; that `i` is not provided by the component instantiating the Pagination component, but rather by the component itself. What I'd really have to do is something like define a `handleSelectPage(i) { this.setPage(i-1) }` method on FormController and pass that to Pagination and that just seems like a silly thing to do for a rule we're not actually even enforcing